import-export-cli/integration/config.yaml

```diff
@@ -14,4 +14,4 @@ indexing-delay: 1000
 
 dcr-version: v0.17
 rest-api-version: v1
-apictl-version: 3.2.0
\ No newline at end of file
+apictl-version: 4.0.0
```

Comment:
Add line EOF

Reply:
This has been fixed recently in [1]. Hence, can ignore this comment.

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/integration/config.yaml